.c8rc-versioned.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Is coverage on these tests actually useful? I get the unit/integration tests, but I feel like there will be very limited coverage on these

Reply:
versioned tests are the only way we get instrumentation coverage.  these are essentially integration tests for all 3rd party libraries.  When we use a code coverage tool all the coverages will be merged together so it'll give a complete picture.  if you run unit/integration tests you'll see the gaping hole in instrumentation.

Reply:
I wasn't thinking about the mushing together of coverage, this makes more sense now, thanks!